src/usr/local/www/services_dyndns_edit.php

```diff
@@ -491,6 +492,9 @@ function setVisible(service) {
 				hideInput('ttl', false);
 				break;
 			case "namecheap":
+			case "cloudflare-v6":
+			case "cloudflare":
```

Comment:
I see - the code in this commit did not have that (I was clicking the View button from here) but in master and RELENG_2_3 the other case got added by a different commit. I guess the necessary merge resolution missed the mark a bit.